logdna/logdna.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, key, options={}):
         self.retry_interval_secs = options.get('retry_interval_secs', defaults['RETRY_INTERVAL_SECS'])
         self.tags = options.get('tags', [])
         self.buf_retention_byte_limit = options.get('buf_retention_limit', defaults['BUF_RETENTION_BYTE_LIMIT'])
+        self.user_agent = 'python/%s' % (options.get('logdna_pkg_v', defaults['LOGDNA_PKG_V']))
```

Comment:
@vilyapilya, let's make it consistent across all code libraries:
```
self.user_agent = options.get('user_agent', defaults['USER_AGENT'])
```
where `defaults['USER_AGENT'] = 'python/%s` %s __version__`

The main reason for this is making this overridable by custom libraries built on top of `python`